File: apps/pwa/src/api/add_music_to_musicbill.ts
import { request, Method } from '.';

function addMusicToMusicbill(musicbillId: string, musicId: string) {
  return request({
    method: Method.POST,
    path: '/api/musicbill_music',
    body: {
      musicbillId,
      musicId,
    },
  });
}

export default addMusicToMusicbill;

File: apps/flutter/lib/server/api/add_music_to_musicbill.dart
import '../request.dart';

Future<void> addMusicToMusicbill({
  required String musicbillId,
  required String musicId,
}) async {
  await httpPost(
    path: "/api/musicbill_music",
    data: {"musicbillId": musicbillId, "musicId": musicId},
    withToken: true,
  );
}
